apps/email/js/sync.js

```diff
@@ -501,7 +503,7 @@ define(function(require) {
 
           sendNotification(
             BACKGROUND_SEND_NOTIFICATION_ID,
-            mozL10n.get('background-send-error-title'),
+            'background-send-error-title',
```

Comment:
so, here you are passing L10n Object, which is proper, but you also have to switch all the `subject = mozL10n.get` to `subject = {id: ..., args... };`.